custom/nsutil/README.md

```diff
@@ -0,0 +1,17 @@
+# NSUtil
+
+NSUtil is a cli tool that can be used to execute commands in target namespaces, very similar to nsenter. This tool also forwards any kill signals to the executed command
+and also pipes the standard input and output from the target command. Currently this doesn not support mount and user namespaces.
+
+## Usage
+
+```sh
+./nsutil -p -n -t 39590 -- <target-command>
+```
+### Flags
```

Comment:
we can also ask to move the binary in `/usr/local/bin` and use the command directly as well?

Reply:
Yes this is just an example of how to use the binary directly